pywbem_mock/_dmtf_schema.py

```diff
@@ -0,0 +1,392 @@
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#
+#
+"""
+A DMTF_Schema is the collection of mof files managed by the DMTF and
+released as a single tested, coherent package with a defined Major, Minor,
+Revision number.  The mof for these files is packaged as a single zip
+file and is available on the DMTF web site.
+
+The DMTF schema is the core set of mofs that represent the models used in
+most CIM/WBEM implementations that that utilize pywbem.
+
+The DMTFSchema class installs a DMTF MOF schema into a directory and makes
+information on that directory availableas a property. Donwloads a DMTF schema
+defined by its version number from the DMTF web site and expands it into a
+directory defined by a constructor input parameter.
+
+The version defined will be downloaded in the directory defined by the
+'schema_dir' constructor parameter if that directory is empty does not exist
+and expanded from the downloaded zip file if the expanded files do not exist.
+
+Installation into the 'schema_dir' directory includes:
+
+1. If necessary, creates the directory defined by schema_dir.
+
+2. Downloads the schema from the DMTF web site into the schema_dir.
+
+3. Expands the zip file for the mof into the directory 'mof<schemaVersionNumber'
+   in the schema_dir.
```

Comment:
This is an overview section, but the text is nearly more specific as the class description. I don't think we need the exact names of parameters, properties and directory names at this point. I think this list here could be reduced to some additions to the paragraph before the one I suggest to be removed, i.e. by adding "if it does not exist already" or so.

Reply:
Cleaned up.

Reply:
DONE